### DISCUSSION TITLE
Download via Civitai url?

### DISCUSSION THREAD

**Discussion by Zaazu** [BlafKing/sd-civitai-browser-plus#62] [Ideas]
Hi, 

Is there any way to just paste in a URL so the extension downloads it, instead of searching?

Cheers

**Comment by BlafKing** [comment]
I'll try to make it so that you can input the URL into the search bar so it'll show the model of the URL that got inserted when clicking search👍🏻 

Sidenote: You can already copy the SHA256 of the model on the civitAI page and input that into the Search box when searching by model name to get the corresponding model as a search result, not a feature made by me, but supported by the CivitAI API :)
![image](https://github.com/BlafKing/sd-civitai-browser-plus/assets/9644716/591a867a-04b9-4169-880e-6dbb6bf00dd3)

**Comment by BlafKing** [comment]
It's now possible to input the URL into the search box in v1.14.5, you can then select the model and download it like normal! :)
